extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -68,6 +70,64 @@ function TrackedCornerstoneViewport({
     };
   }, []);
 
+  useEffect(() => {
+    const { unsubscribe } = MeasurementService.subscribe(
+      MeasurementService.EVENTS.JUMP_TO_MEASUREMENT,
+      ({ viewportIndex: jumpToMeasurementViewportIndex, measurement }) => {
+        // check if the correct viewport index.
+        // TODO -> Jump to measurement implementation.
+
+        if (viewportIndex !== jumpToMeasurementViewportIndex) {
+          // Event for a different viewport.
+          return;
+        }
+
+        if (
+          measurement.displaySetInstanceUID !== displaySet.displaySetInstanceUID
```

Comment:
Wouldn't this update the set displaySet? Or are we having the ViewportGridService reconcile this bit, and then on `create/propChange` we check the jump cache to see if we have one pending?

If it's the latter, where's that jump cache check?

Reply:
The `ViewportGrid` calls the `ViewportGridService` and changes the layout.
In the meantime if we have a different viewport that can't process the jumpToMeasurement event, it exits early here. The viewport that is replaced later will then consume it.